packages/fuels/package.json

```diff
@@ -55,7 +55,8 @@
     "build:package": "tsup",
     "build:browser": "pnpm vite build",
     "build:minified": "pnpm uglifyjs --compress --mangle --output dist/browser.min.mjs -- dist/browser.mjs",
-    "postbuild": "run-s type:declarations",
+    "postbuild": "run-s build:recipes type:declarations",
+    "build:recipes": "cd ../recipes && pnpm build:all && cd ../fuels",
```

Comment:
The plan is to remove the `fuels-typegen` command and only have `fuels typegen`, but this caused failures in the recipes which were using `fuels-typegen` because `@fuel-ts/recipes` gets built _before_ `fuels` and so the `fuels typegen` command is unavailable. This is a workaround which ensures that everything is still built as before but with the `fuels typegen` command. cc @danielbate

Reply:
I'm not sure I'm aware of such a plan.

Reply:
Sorry, bad wording; the idea is to remove `fuels-typegen` because I don't see it as necessary to have two commands doing the same thing.

Reply:
The `fuels` CLI is an umbrella for the other CLIs, so it's safer to keep it as is and just follow whatever is in place.

Reply:
Whatever is in place for typegen, I mean.